### PR TITLE
Add `jira point` as a command

### PR DESCRIPTION
**Pull request by robustdj**
This PR adds the command to add story points to a given ticket in JIRA with a command like

```
jira point GROW-123 as 5
```

### CODE REVIEW

File: lib/lita/handlers/jira.rb [hunk 1]
```diff
@@ -90,6 +99,14 @@ def comment(response)
         response.reply(t('comment.added', issue: issue.key))
       end
 
+      def point(response)
+        issue = fetch_issue(response.match_data['issue'])
+        return response.reply(t('error.request')) unless issue
+        points = response.match_data['points']
+        issue.save(fields: { customfield_10004: points.to_i }) # Story points is a custom field with id 10004
```
Comment (robustdj) [inline lib/lita/handlers/jira.rb]: I just realized this is unique to our JIRA setup. Any thoughts on how to parameterize this?

Comment (robustdj) [inline lib/lita/handlers/jira.rb]: Perhaps a line in `lita_config.rb`?

File: locales/en.yml [hunk 2]
```diff
@@ -19,6 +20,9 @@ en:
           comment:
             syntax: jira comment on <issue> <comment text>
             desc: Adds <comment text> to <issue>
+          point:
+            syntax: jira point <issue> as <points>
+            desc: Adds <points> Story Points to <issue>
```
Comment (tristaneuan) [inline locales/en.yml]: Is the Story Points field specific to our JIRA organization? If so, you may want to change this template to something more generic like "points"

File: locales/en.yml [hunk 3]
```diff
@@ -19,6 +20,9 @@ en:
           comment:
             syntax: jira comment on <issue> <comment text>
             desc: Adds <comment text> to <issue>
+          point:
```
Comment (tristaneuan) [inline locales/en.yml]: There seems to be a mismatch here between the translation namespace defined in this file (`point`) and the one you're using to reference it in the handler code (`points`)

**Comment by tristaneuan** [comment]
Happy path seems to work fine, but you may want to catch the error thrown when the field doesn't exist on a particular and notify the user:
```Cadet URL > ~jira point DEV-824 as 5
[2016-05-31 17:33:09 UTC] DEBUG: Dispatching message to Lita::Handlers::Jira#point.
>>>>>>>>> Exception response: {"errorMessages":[],"errors":{"customfield_10004":"Field 'customfield_10004' cannot be set. It is not on the appropriate screen, or unknown."}}
Added a point estimation of 5 to DEV-824```

File: lib/lita/handlers/jira.rb [hunk 4]
```diff
@@ -149,6 +147,17 @@ def invalid_ambient?(response)
...
         config.ignore.include?(user.id) || config.ignore.include?(user.mention_name) || config.ignore.include?(user.name)
       end
+
+      def set_points_on_issue(issue, response)
+        points = response.match_data['points']
+        begin
+          issue.save!(fields: { config.points_field.to_sym => points.to_i })
+        rescue
+          response.reply(t('error.unable_to_point'))
+          return
```
Comment (tristaneuan) [inline lib/lita/handlers/jira.rb]: This can be shortened to `return response.reply(t('error.unable_to_point'))`

**Comment by tristaneuan** [comment]
Looks great, thanks for the contribution!

**Comment by tristaneuan** [comment]
@esigler I've pushed the v0.8.1 tag, but I don't have permission to release this on rubygems.org. Could you do the honors?

**Pull request merged**